specs/merkle-tree.md

```diff
@@ -147,6 +145,8 @@ Outputs a [SphinxMerkleTree](https://github.com/sphinx-labs/sphinx/blob/develop/
 ## Dependencies
 The Merkle tree generation utility makes calls to external libraries. We use `ethers` version 6.7.0 to handle ABI encoding tree leaf `data` and `@openzeppelin/merkle-tree` version 1.0.5 to assemble the Merkle tree. We test that interactions with these libraries work correctly in our [SphinxModuleProxy tests](https://github.com/sphinx-labs/sphinx/blob/develop/packages/contracts/test/SphinxModuleProxy.t.sol) which test that Merkle trees generated by this logic have data that is encoded correctly and that they are executable on-chain. However, we do not test the internals of these libraries and instead rely on the assumption that they are robust and bug-free.
 
+Specifically, the Sphinx Merkle tree generation logic relies on the `@openzeppelin/merkle-tree` dependency to [sort](https://github.com/OpenZeppelin/merkle-tree/blob/0951d3ca036d44c61e33fd9731750cf0ce2978c3/src/standard.ts#L39-L43) the tree leaves in descending order by their [hash](https://github.com/OpenZeppelin/merkle-tree/blob/0951d3ca036d44c61e33fd9731750cf0ce2978c3/src/standard.ts#L9). This is required to satisfy [invariant 9](#9-merkle-tree-leaves-must-be-ordered-in-the-tree-by-the-leafs-hash-descending). We assume that their hashing and sorting logic is robust and bug-free.
```

Comment:
Do these link to the version of the OpenZeppelin Merkle tree library that we actually use? (1.0.5)

Reply:
Good catch, updated to point to the specific commit